### ISSUE TITLE
Upgraded IDE and now I'm seeing wierd ligature overlapping.

### ISSUE THREAD

**Issue by Orillian** [kiliman/operator-mono-lig#109]
I love all the new ligatures and the latest builds have been working great for me with everything... that is until the last couple days when I upgraded some of my IntelliJ Software.

For example: After updating phpStorm to v2019.2 my ligatures look like the following.
![overlap1](https://user-images.githubusercontent.com/823192/65347259-e0d54d80-db9b-11e9-8a8a-6ba625d4a013.jpg)

I've expanded this to 16pt so it's a bit easier to see, but with this font only the above is what happens! :(

This is happening with so far both of my IntelliJ IDE's I've updated to v2019.x

I've tried Fira and a couple others and their ligatures have proper spacing after them.

I was wondering if anyone else was experiencing this issue?

**Comment by Orillian** [comment]
I'm not blaming op-mono-lig for this, but I'm curious as to why it's the only font I have with Ligatures that this happens to? I'm sure phpStorm has changed how they display ligatures in one of the updates between 2018 and 2019, but I have no clue what that change might be. I've been looking and so far can't find anything. 

Also of note, this only happens to the ligatures. I'm not seeing bad spacing between other glyphs in the font.

**Comment by markypython** [comment]
What platform are you using (macOS, Windows, etc.)?

**Comment by kiliman** [comment]
Hmm... I don't really use IntellJ, but I do Android Studio on Mac. Here's what I see with the latest version.

![image](https://user-images.githubusercontent.com/47168/65348587-8e059100-dbaf-11e9-8ba3-fd84028ec03d.png)

**Comment by kiliman** [comment]
Do you have spaces after your ligature? It should work either way, but just wondering.

Also, which font are you using? Weight + SSm or not?

**Comment by Orillian** [comment]
I'm on Windows 10, latest release on two devices both running PhpStorm and PyCharm Community Edition and the 1803 release on the third only using PhpStorm on this third machine. 

Note this started happening when I upgraded to the 2019.x versions of these IDE's. 
It happens using both the older (built a year ago version of the font) **Operator Mono Lig Book** as well as one I built using the new release just recently.

On a side note I'm still using an older version of Android Studio, have not updated it yet and everything works fine in that application, as well as in VS Code.

As to the spacing, I add an extra space after each ligature and it looks like your image above.

**Comment by kiliman** [comment]
I installed PhpStorm 2019.2 on Windows 10.

Here is Operator Mono Lig Light (we don't have Book for v2). So it looks like it should be working. Send me your email to kiliman@gmail.com so we can troubleshoot. Thanks!

![image](https://user-images.githubusercontent.com/47168/65436298-f17e0180-ddef-11e9-9d4e-f691ea359a79.png)

**Comment by kufii** [comment]
try rebuilding your font, that fixed things for me

**Comment by Orillian** [comment]
Ok, I'm feeling a bit silly. 1.0 fonts are the ones that no longer work for me on 2019.2 builds of IntelliJ IDE's.

I managed to read the list of fonts a dozen times or more and completely missed the fact that Book which was available in 1.0 was NOT part of the 2.0 build. 
So when I updated to 2019.2 and my 1.0 ligatures broke I decided it was time to try the 2.x ones out. I ran the build and installed the fonts found in the build folder, apparently I didn't look close enough to realize the ones I used were now missing.

That said Light Lig does indeed work for me...so that is good at least!

On a side note even after attempting to rebuild Book with the 1.0 builder I cannot get it working with 2019.2.

I guess I'll have to bump my font sizes and try using light lig until Book gets redone!
